File: myblog/src/main/java/myblog/domain/article/dto/response/ArticleDetailResDto.java
package myblog.domain.article.dto.response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import myblog.domain.article.entity.Article;

import java.time.LocalDateTime;

@Builder
@Setter
@Getter
public class ArticleDetailResDto {
    private Long id;
    private String title;
    private String content;
    private String writer;
    private LocalDateTime createdAt;
    private Long viewCount;
    private Long startCount;

    public static ArticleDetailResDto toDto(Article article) {
        return ArticleDetailResDto
                .builder()
                .id(article.getId())
                .title(article.getTitle())
                .content(article.getContent())
                .writer(article.getWriter())
                .createdAt(article.getCreatedAt())
                .viewCount(article.getViewCount())
                .startCount(article.getStarCount())
                .build();
    }
}

File: myblog/FE/front/src/dto/article/response/ArticleDetailResDto.js
class ArticleDetailResDto {

    id = 0;
    title = "";
    content = "";
    writer = "";
    createdAt = "";
    viewCount = 0;
    starCount = 0;

    constructor(id, title, content, writer, createdAt, viewCount, starCount) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.writer = writer;
        this.createdAt = createdAt;
        this.viewCount = viewCount;
        this.starCount = starCount;
    }
    
}

export default ArticleDetailResDto;